client/src/app/site/users/components/user-list/user-list.component.ts

```diff
@@ -365,9 +365,16 @@ export class UserListComponent extends ListViewBaseComponent<ViewUser, User> imp
      * Handler for bulk setting new passwords. Needs multiSelect mode.
      */
     public async resetPasswordsSelected(): Promise<void> {
-        for (const user of this.selectedRows) {
+        if (this.selectedRows.find(row => row.user.id === this.operator.user.id)) {
+            this.raiseError(
+                this.translate.instant(
+                    'Your own password cannot be reset here. Please use the password change dialog instead.'
```

Comment:
`Note: You password will be excluded. Please use the password change dialog instead.`